examples/example4/main.go

```diff
@@ -0,0 +1,54 @@
+package main
```

Comment:
Please add identical comment above this example as with other's in example folder:
```go 
// Copyright 2012 The GoSNMP Authors. All rights reserved.  Use of this
// source code is governed by a BSD-style license that can be found in the
// LICENSE file.
```